#!/usr/bin/python3

def raise_exception_msg(message=""):
    try:
        2 * boy
    except:
        print("{}".format(message))
